front/lib/resources/data_source_resource.ts

```
@@ -297,6 +276,18 @@ export class DataSourceResource extends ResourceWithVault<DataSourceModel> {
     });
   }
 
+  static async fetchByIds(
+    auth: Authenticator,
+    ids: string[],
+    options?: Omit<FetchDataSourceOptions, "limit" | "order">
+  ) {
+    return DataSourceResource.fetchByModelIds(
+      auth,
+      removeNulls(ids.map(getResourceIdFromSId)),
```

Q: We want to silently ignore when we send sIds that are not valid?